go-controller/pkg/util/net_linux.go

```diff
@@ -170,15 +170,31 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	return false, nil
 }
 
-func DeleteConntrack(ip string) {
+func DeleteConntrack(ip string, ports []int32) {
 	ipAddress := net.ParseIP(ip)
 	if ipAddress == nil {
 		klog.Errorf("Value \"%s\" passed to DeleteConntrack is not an IP address", ipAddress)
 		return
 	}
 
 	filter := &netlink.ConntrackFilter{}
-	err := filter.AddIP(netlink.ConntrackReplyAnyIP, ipAddress)
+	// 17 = UDP protocol
+	err := filter.AddProtocol(17)
+	if err != nil {
+		klog.Warningf("Could not add Protocol UDP to conntrack filter %v", err)
+		return
+	}
+	for _, port := range ports {
```

Comment:
the filter does an AND with all the fields, this will try to mach a flow with the same port in `orig` and `dest`
I think that we need to execute a `netlink.ConntrackDeleteFilter` per each port

Reply:
done